app/Http/Controllers/Front/PageController.php

```diff
@@ -539,49 +540,68 @@ public function getTemplateOne($helpdesk_products, $trasform)
         try {
             $template = '';
             $temp_controller = new TemplateController();
-            if (count($helpdesk_products) > 0) {
+
```

Comment:
Try this here public function getTemplateOne($helpdeskProducts)
{
    try {
        if ($helpdeskProducts->isEmpty()) {
            return ''; // Return early if no products
        }

        // Prepare arrays for storing computed values
        $productIds = $helpdeskProducts->pluck('id')->toArray();
        $productNames = $helpdeskProducts->pluck('name')->toArray();

        // Fetch highlighted products
        $highlightedProducts = Product::whereIn('name', $productNames)
            ->pluck('highlight', 'name')
            ->toArray();

        // Initialize arrays for storing computed values
        $leastAmounts = [];
        $yearlyAmounts = [];
        $priceDescriptions = [];
        $monthPriceDescriptions = [];

        // Populate arrays with computed values for each product
        foreach ($helpdeskProducts as $product) {
            $productId = $product->id;

            $leastAmounts[$productId] = $this->tempController()->leastAmount($productId);
            $yearlyAmounts[$productId] = $this->YearlyAmount($productId);
            $priceDescriptions[$productId] = $this->getPriceDescription($productId);
            $monthPriceDescriptions[$productId] = $this->getmonthPriceDescription($productId);
        }

        // Transform helpdesk products
        $trasform = $helpdeskProducts->map(function ($product) use ($highlightedProducts, $leastAmounts, $yearlyAmounts, $priceDescriptions, $monthPriceDescriptions) {
            $productId = $product->id;
            $productName = $product->name;
            $highlight = $highlightedProducts[$productName] ?? false;

            return [
                'price' => $leastAmounts[$productId],
                'price-year' => $yearlyAmounts[$productId],
                'price-description' => $priceDescriptions[$productId],
                'pricemonth-description' => $monthPriceDescriptions[$productId],
                'strike-price' => $leastAmounts[$productId], // Not sure if this is intended to be different
                'strike-priceyear' => $yearlyAmounts[$productId],
                'name' => $productName,
                'feature' => $product->description,
                'subscription' => $product->type == 4 ? '' : $this->tempController()->plans($product->shoping_cart_link, $productId),
                'url' => $product->add_to_contact != 1 ?
                    ($product->type == 4 ?
                        '<button class="btn ' . ($highlight ? 'btn-primary' : 'btn-dark') . ' btn-modern buttonsale" data-toggle="modal" data-target="#tenancy" data-mydata="' . $productId . '">
                            <span style="white-space: nowrap;">Order Now</span>
                        </button>' :
                        '<input type="submit" value="Order Now" class="btn ' . ($highlight ? 'btn-primary' : 'btn-dark') . ' btn-modern buttonsale"></form>'
                    ) :
                    '<a class="btn ' . ($highlight ? 'btn-primary' : 'btn-dark') . ' btn-modern sales buttonsale" href="https://www.faveohelpdesk.com/contact-us/">Contact Sales</a>',
            ];
        })->toArray();

        // Fetch template data and transform template
        $data = PricingTemplate::findOrFail(1)->data;
        return $this->transformTemplate('cart', $data, $trasform);
    } catch (\Exception $ex) {
        app('log')->error($ex->getMessage());
        return redirect()->back()->with('fails', $ex->getMessage());
    }
}

Also product type = 4 is not the way to check if it s cloud product or not I have created an helper function for this